proto/server.proto

```diff
@@ -26,6 +26,8 @@ message InfoRes {
   bool dns_enabled = 8;
   string dns_address = 9;
   string filename = 10;
-  bool inactive_enabled = 11;
-  google.protobuf.Duration inactive_duration = 12;
+  string client_config_dns_servers = 11;
+  string client_config_dns_search_domain = 12;
+  bool inactive_enabled = 13;
```

Comment:
New proto fields must have an incremented id, otherwise we break backwards compatibility. Hence, it should be

```proto
bool inactive_enabled = 11;
google.protobuf.Duration inactive_duration = 12;
string client_config_dns_servers = 13;
string client_config_dns_search_domain = 14;
```

See https://protobuf.dev/programming-guides/dos-donts/#dont-re-use-a-tag-number

> Don’t Re-use a Tag Number
Never re-use a tag number. It messes up deserialization. Even if you think no one is using the field, don’t re-use a tag number. If the change was live ever, there could be serialized versions of your proto in a log somewhere. Or there could be old code in another server that will break.

And also https://protobuf.dev/programming-guides/proto3/#assigning-field-numbers and https://protobuf.dev/programming-guides/proto3/#updating

Reply:
:+1: